### PR TITLE
[Enhancement] Create lake table by default if use_staros is true

### PR DESCRIPTION
**Pull request by sduzh**
Signed-off-by: sduzh <zhuming9011@qq.com>

## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto backported to target branch
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3
  - [ ] 2.2

### CODE REVIEW

**Comment by sduzh** [comment]
run starrocks_fe_unittest

**Comment by sduzh** [comment]
run starrocks_fe_unittest

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 20 / 22 (90.91%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/scheduler/mv/IMTCreator.java|0|1|00.00%|[235]|
|:large_blue_circle:|com/starrocks/server/GlobalStateMgr.java|0|1|00.00%|[2198]|
|:large_blue_circle:|com/starrocks/sql/parser/AstBuilder.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/analyzer/CreateTableAnalyzer.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/common/EngineType.java|12|12|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/ast/CreateTableStmt.java|4|4|100.00%|[]|

File: fe/fe-core/src/main/java/com/starrocks/sql/ast/CreateTableStmt.java [hunk 1]
```diff
@@ -152,18 +122,8 @@ public CreateTableStmt(boolean ifNotExists,
...
-            this.engineName = DEFAULT_ENGINE_NAME;
-        } else {
-            this.engineName = engineName;
-        }
-
-        if (Strings.isNullOrEmpty(charsetName)) {
-            this.charsetName = DEFAULT_CHARSET_NAME;
-        } else {
-            this.charsetName = charsetName;
-        }
```
Comment (sduzh) [inline fe/fe-core/src/main/java/com/starrocks/sql/ast/CreateTableStmt.java:165]: `CreateTableAnalyzer.analyze()` will set default engine and default charset.

**Review by wyb — APPROVED** [review]

**Review by gengjun-git — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Pull request merged**